control-plane-operator/controllers/hostedcontrolplane/hostedcontrolplane_controller.go

```diff
@@ -2572,31 +2579,6 @@ func (r *HostedControlPlaneReconciler) reconcileOperatorLifecycleManager(ctx con
 		}); err != nil {
 			return fmt.Errorf("failed to reconcile catalog rollout rolebinding: %w", err)
 		}
-
-		certifiedOperatorsCronJob := manifests.CertifiedOperatorsCronJob(hcp.Namespace)
```

Comment:
Also realized... maybe not as important, but maybe we should have code to delete these jobs if they exist.

Reply:
I thought about this but we aren't GA yet, and they just cause a little hourly pod churn.  New clusters won't have them.  I thought we could just let them linger for old non-GA clusters.  But if anyone else feels strongly, I'm up for changing my mind.